### PR TITLE
Daily plan

### PR DESCRIPTION
**Pull request by vovopap**

### CODE REVIEW

File: translations/README-uz.md [hunk 1]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
 - Javascript
 - HTML, CSS, and other front-end technologies
 
-## The Daily Plan
+## Kunlik plan
```
Comment (Hojarbu) [inline translations/README-uz.md]: reja

File: translations/README-uz.md [hunk 2]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
+Bazi mavzular bir kun oladi, va bazilariga bir necha kunlar kerak bo'ladi. Bazilari faqat o'rganish, kod yozish kerak emas.
 
-Each day I take one subject from the list below, watch videos about that subject, and write an implementation in:
-- C - using structs and functions that take a struct * and something else as args
-- C++ - without using built-in types
-- C++ - using built-in types, like STL's std::list for a linked list
-- Python - using built-in types (to keep practicing Python)
-- and write tests to ensure I'm doing it right, sometimes just using simple assert() statements
-- You may do Java or something else, this is just my thing
+Har kuni bitta mavzu olib, videolarini ko'rib, quyidagi tillarda kodda yozib chiqiman:
```
Comment (Hojarbu) [inline translations/README-uz.md]: chiqaman

File: translations/README-uz.md [hunk 3]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
+- C - struct-lardan va struct * oladigan funksiyalarni ishlatgan xolda
```
Comment (Hojarbu) [inline translations/README-uz.md]: holda

File: translations/README-uz.md [hunk 4]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
+- C++ - built-in tiplarni ishlatmagan xolda
```
Comment (Hojarbu) [inline translations/README-uz.md]: xolda -holda, hamma joyda

File: translations/README-uz.md [hunk 5]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
-You don't need all these. You need only [one language for the interview](#pick-one-language-for-the-interview).
+Sizga bularni hammasi kerak emas. Intervyu uchun sizga [faqat bitta til](#pick-one-language-for-the-interview) kerak.
 
-Why code in all of these?
-- Practice, practice, practice, until I'm sick of it, and can do it with no problem (some have many edge cases and bookkeeping details to remember)
-- Work within the raw constraints (allocating/freeing memory without help of garbage collection (except Python or Java))
-- Make use of built-in types, so I have experience using the built-in tools for real-world use (not going to write my own linked list implementation in production)
+Nega turlicha yozish kerak?
+- Mashq, mashq, mashq, ko'nglim ayniguncha. Shunda muammosiz tushimda ham qila olaman.
+- Pastgi darajada (low level) yoza olish uchun (masalan, garbage collection ishlatmagan xolda)
```
Comment (Hojarbu) [inline translations/README-uz.md]: holda

File: translations/README-uz.md [hunk 6]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
+- Har kunlik ishda built-in tiplardan bemarol foydalana olish uchun (production-da array listni yoki linked listni boshidan yozib o'tirmaymanku)
```
Comment (Hojarbu) [inline translations/README-uz.md]: bemalol

File: translations/README-uz.md [hunk 7]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
 
-I may not have time to do all of these for every subject, but I'll try.
+Har bir mavzuni tepadagi tarzda o'qiyolmashim mumkin, lekin harakat qilaman.
```
Comment (Hojarbu) [inline translations/README-uz.md:475]: Har bir mavzuni yuqoridagi kabi yoritib bera olmasligim mumkin, lekin harakat qilaman.

File: translations/README-uz.md [hunk 8]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
 
-You can see my code here:
+Meni kodlarimni bu yerda ko'rishingiz mumkin:
  - [C](https://github.com/jwasham/practice-c)
  - [C++](https://github.com/jwasham/practice-cpp)
  - [Python](https://github.com/jwasham/practice-python)
 
-You don't need to memorize the guts of every algorithm.
+Har bir algoritmni ich-kirini eslab qolishingiz shart emas.
```
Comment (Hojarbu) [inline translations/README-uz.md]: ich-kirini -- ikr-chikirini demoqchimidiz

File: translations/README-uz.md [hunk 9]
```diff
@@ -447,35 +447,35 @@ These are prevalent technologies but not part of this study plan:
...
 
-Write code on a whiteboard or paper, not a computer. Test with some sample inputs. Then test it out on a computer.
+Birinchi kodizni doskada yoki qog'ozda yozing, komputerda emas. Og'zaki test qiling. Keyin komputerda yozib test qiling.
```
Comment (Hojarbu) [inline translations/README-uz.md]: kompYuterda

File: translations/README-uz.md (hunk 6 above)
Comment (vovopap) [reply translations/README-uz.md]: eshitganimni yozdimda :))

File: translations/README-uz.md (hunk 7 above)
Comment (vovopap) [reply translations/README-uz.md:475]: bu yerda yoritish xaqida gap ketmayapti. shunaqa tarzda uqish haqida ketyapti. you can consider this whole project as him writing his thoughts as notes.

File: translations/README-uz.md (hunk 8 above)
Comment (vovopap) [reply translations/README-uz.md]: yeah. that is embarrassing :(

**Pull request merged**